js/chart.js

```diff
@@ -0,0 +1,220 @@
+// Filter out an array with the users in past 60 minutes
+function filter(data) {
```

Comment:
i am just a rando person, but strongly advise against calling anything `filter`. That is not a useful name. Call this `filterUsersPastHour` etc and the name will be more clear plus the comment will not be required.